### DISCUSSION TITLE
🔗 Allow paying premium users / agencies to disable the nag screens of Elementor (Duplicate of #22796)

### DISCUSSION THREAD

**Discussion by mbaierl** [elementor/elementor#27354] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Elementor nags my users about other premium plugins, the AI tools etc.

### Describe the solution you'd like.

Add filters and hooks so developers (Pro users = the ones already paying for your tools) to be able to disable all (!) nag screens.

As agency our users do not ned to be informed about any potential upsell of Elementor Pro.

### Describe alternatives you've considered.

Since there are no hooks or filters right now in the Hints API the only other solution is switching to Bricks, which does not have any nag screens.

### Additional context

See https://github.com/elementor/elementor/issues/25739

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by hitsgee** [comment]
Yes please! The intrusion is causing great concern for my clients.

It shouldn't be called pro and have a price tag with all this new bloat introduced.

**Comment by nicholaszein** [comment]
Hi @mbaierl!

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

### 🔗 🚀 There is already a similar request submitted by someone here #22796

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

Kind regards
